### DISCUSSION TITLE
Adding Table inside Resource Form

### DISCUSSION THREAD

**Discussion by Professor93M** [filamentphp/filament#11395] [Help]
Hi friends,

Please are there any way to adding table of information and Radio, returns ID of selected row with the submission Form.

**Comment by leandrocfe** [comment]
Sorry, I didn't understand the question

**Reply by Professor93M** [reply]
I have a Form, I want to show with this form a table
This table must have a radio input to select the row that must be return the ID of the selected row after submission

**Reply by leandrocfe** [reply]
Custom livewire component I think
